msresist/clustering.py

```diff
@@ -19,12 +19,13 @@ class MassSpecClustering(GaussianMixture):
     """ Cluster peptides by both sequence similarity and data behavior following an
     expectation-maximization algorithm. SeqWeight specifies which method's expectation step
     should have a larger effect on the peptide assignment. """
-    def __init__(self, info, ncl, SeqWeight, distance_method):
-        super().__init__(n_components=ncl, covariance_type="diag", n_init=2, max_iter=200, tol=1e-4)
+    def __init__(self, info, ncl, SeqWeight, distance_method, random_state=None):
+        super().__init__(n_components=ncl, covariance_type="diag", n_init=2, max_iter=200, tol=1e-4, random_state=random_state)
 
         self.info = info
         self.SeqWeight = SeqWeight
         self.distance_method = distance_method
+        self.random_state = random_state
```

Comment:
Setting the argument `random_state` to an integer makes DDMC to be reproducible (whereas `None` makes the seeding random) so I added this argument. I figured that could be useful to easily switch. @aarmey

Reply:
Excellent—yes!